Description: Remove binary code from the package
 Upstream ships a binary plugin in the archive. This plugin is not necessary to
 get PlayOnLinux working and it still can be downloaded from upstream.
Forwarded: not-needed
Author: Bertrand Marc <beberking@gmail.com>
--- a/plugins/plugins.lst
+++ b/plugins/plugins.lst
@@ -21,9 +21,6 @@
 
 ## in fact empty line is also ignored because it's more easy to read =)
 
-##Capture
-capture;Capture_2.4.3.pol;Capture;enable;2.4.3;11;1;0
-
 ##Screencap
 screencap;ScreenCap_0.0.4.pol;ScreenCap;enable;0.0.4;1;1;1
 
